### DISCUSSION TITLE
Add data to default error response

### DISCUSSION THREAD

**Discussion by samuelclark907** [pydantic/pydantic#4068] [Question]
I would like to keep the default error message response. I only need to add a request-id to the response. Is there a way to do this without creating a separate custom error handler?

current response:

[
  {
    "loc": [
      "mbr_idn"
    ],
    "msg": "Perameter length is invalid",
    "type": "value_error"
  }
]

wanted response:

[
  {
    "loc": [
      "mbr_idn"
    ],
    "msg": "Perameter length is invalid",
    "type": "value_error",
    "requestid": "dasda-324f-dsf3"
  }
]

**Comment by samuelcolvin** [comment]
Please format your questions properly.

You can just edit the list returned from `validation_error.errors()`.
